collections/_release_4_5/entry-general-gui-handle-complex-gui-easily-with-recursive-overrides.md

```diff
@@ -0,0 +1,22 @@
+---
+type: entry
+section: general
+subsection: GUI
+rank: 2
+importance: 3
+anchor: handle-complex-gui-easily-with-recursive-overrides
+title: Handle complex GUI easily with recursive overrides
```

Comment:
```suggestion
title: Handle complex GUIs easily with recursive overrides
```

Reply:
I would keep the singular, or use "Handle complex UI easily" (we use UI for this in the documentation etc.)